src/evergreen/api.py

```diff
@@ -61,9 +61,22 @@
 
 CACHE_SIZE = 5000
 DEFAULT_LIMIT = 100
-MAX_RETRIES = 3
-START_WAIT_TIME_SEC = 2
-MAX_WAIT_TIME_SEC = 5
+
+DEFAULT_HTTP_RETRY_ATTEMPTS = 10
```

Comment:
I decided to align the backoff and retries default values with the [urllib3 default ones](https://urllib3.readthedocs.io/en/stable/reference/urllib3.util.html#urllib3.util.Retry.DEFAULT_ALLOWED_METHODS) because they seemed much more reasonable.